gammapy/data/event_list.py

```diff
@@ -379,6 +380,45 @@ def filter_circular_region(self, region):
 
         return mask
 
+    def select_region(self, region, wcs):
+        """Select events in circular regions.
+
+        Parameters
+        ----------
+        region : `~regions.SkyRegion` or str
+            Sky region or string defining a sky region
+        wcs : `~astropy.wcs.WCS`
+            the world coordinate system transformation to assume
+
+        Returns
+        -------
+        event_list : `EventList`
+            Copy of event list with selection applied.
+        """
+        mask = self.filter_region(region, wcs)
+        return self.select_row_subset(mask)
+
+    def filter_region(self, region, wcs):
+        """Create selection mask for event in given circular regions.
+
+        Parameters
+        ----------
+        region : `~regions.SkyRegion` or str
+            Sky region or string defining a sky region
+        wcs : `~astropy.wcs.WCS`
+            the world coordinate system transformation to assume
+
+        Returns
+        -------
+        index_array : `numpy.ndarray`
+            Index array of selected events
+        """
+        region = make_region(region)
+        position = self.radec
+        mask = np.where(region.contains(position, wcs))[0]
```

Comment:
I think there is no need for the call `np.where()`, just remove and remove a boolean array. Or is there an advantage in having the indices?

Reply:
Right.
In reality, there is probably no need to have `filter_region` and `select_region` at the same time.  This is copied from the `filter_circular_region` and `select_circular_region` logic. But this is probably irrelevant. `filter_circular_region` is never used in the code.